amtl/am-enum.h

```diff
@@ -45,32 +45,32 @@ namespace ke {
 } // namespace ke
 
 #define KE_DEFINE_ENUM_OPERATORS(EnumName)                                          \
-  static inline EnumName operator |(const EnumName &left, const EnumName &right) {  \
+  static inline EnumName operator |(const EnumName& left, const EnumName& right) {  \
     typedef ke::enum_integral_type<EnumName>::type int_type;                        \
     return EnumName(int_type(left) | int_type(right));                              \
   }                                                                                 \
-  static inline EnumName operator &(const EnumName &left, const EnumName &right) {  \
+  static inline EnumName operator&(const EnumName& left, const EnumName& right) {  \
```

Comment:
Probably a nit, but the script looks like it made `operator &` inconsistent with other operator definitions.

Reply:
There are more cases of this. Same for `operator *`.